tests/testthat/test_changeMissings.R

```diff
@@ -73,12 +76,28 @@ test_that("sequence bug with labeled variables", {
                          var1 = c(0, 1, 1, 1, -99, -98, -97))
   dat1 <- import_DF(dat1_seq)
 
-  dat1_relabel <- changeValLabels(dat1, varName = "var1", value = c(0, 1, -99, -97), valLabel = c("true",
-                                                                                                  "false",
-                                                                                                  "don't know",
-                                                                                                  "no idea"))
+  dat1_relabel <- changeValLabels(dat1, varName = "var1", value = c(0, 1, -99, -97),
+                                  valLabel = c("true", "false", "don't know", "no idea"))
 
 
-  dat1_changed <- changeMissings(dat1_relabel, varName = "var1", value = c(-99:-97), missings = rep("miss", 3))
+  dat1_changed <- changeMissings(dat1_relabel, varName = "var1", value = c(-99:-97),
+                                 missings = rep("miss", 3))
   expect_equal(dat1_changed$labels[2:4, "missings"], rep("miss", 3))
 })
+
+test_that("changeMissings for multiple variables at once",{
+  out <- changeMissings(dfSAV, varName = c("VAR1", "VAR2"), value = c(1, 2),
+                        missings = c("miss", "valid"))
+
+  expect_equal(nrow(out$labels[out$labels$varName == "VAR1", ]), 4)
+  expect_equal(out$labels[3, "missings"], "miss")
+  expect_equal(out$labels[4, "missings"], "valid")
+  expect_equal(out$labels[3, "value"], 1)
+  expect_equal(out$labels[4, "value"], 2)

```

Comment:
Nit: I have struggled to determine which values would be in lines 3 and 4. Maybe another way to write this could be:

```r
  out_split1 <- out$labels[out$labels$varName == "VAR1", ]
  expect_equal(nrow(out_split1), 4)
  expect_equal(out_split1[out_split1$value == 1, "missings"], "miss")
  expect_equal(out_split1[out_split1$value == 2, "missings"], "valid")
```

Reply:
Mostly agree and adopted.